Assets/Mirage/Runtime/ServerObjectManager.cs

```diff
@@ -294,19 +291,19 @@ void Respawn(NetworkIdentity identity)
             }
         }
 
-        internal bool InternalReplacePlayerForConnection(INetworkPlayer player, INetworkClient client, GameObject character, bool keepAuthority)
+        internal void InternalReplacePlayerForConnection(INetworkPlayer player, INetworkClient client, GameObject character, bool keepAuthority)
         {
             NetworkIdentity identity = character.GetComponent<NetworkIdentity>();
             if (identity is null)
             {
                 logger.LogError("ReplacePlayer: playerGameObject has no NetworkIdentity. Please add a NetworkIdentity to " + character);
-                return false;
+                return;
```

Comment:
can this be exception instead of log

Reply:
fixed

Reply:
they are both exceptions for AddPlayer. should they both be here?

Reply:
fixed